resources/sync_gateway_configs/sync_gateway_default_functional_tests_no_port_cc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "interface":":4984",
+    "adminInterface": "0.0.0.0:4985",
+    "maxIncomingConnections": 0,
+    "maxCouchbaseConnections": 16,
+    "maxFileDescriptors": 90000,
+    "slowServerCallWarningThreshold": 500,
+    "compressResponses": false,
+    "log": ["CRUD+", "Cache+", "HTTP+", "Changes+", "Import+"],
+    "databases":{
+        "db":{
+            {{ autoimport }}
+            {{ xattrs }}
+            "server":"{{ server_scheme }}://{{ couchbase_server_primary_node }}",
```

Comment:
Why doesn't this need to specify the rest of the conditional properties seen in the above files (use_views, num_index_replicas, etc)?

Reply:
I have added missing ones now. Yeah I did not add when I sent PR, I added later while triaging failures on jenkins